src/inmanta/agent/agent.py

```diff
@@ -372,40 +372,58 @@ class DeployRequestAction(str, enum.Enum):
     """ interrupt the current deploy: cancel it, start the new one and restart the current deploy when the new one is done"""
 
 
+# Two letter abbreviations to make table line up
+# N normal
+# P periodic
+# F full
+# I incremental
+NF = (True, False)
+NI = (False, False)
+PF = (True, True)
+PI = (False, True)
```

Comment:
I like naming them, though I would prefer non-abbreviated names: `normal_full`, `periodic_increment`, ... I think they are sufficiently concise that it would only help readability. Now I find myself constantly having to translate letters into words into concepts. No strong preference, I find I do get used to it after a few minutes.

Reply:
The problem is that I need 
1. short names or black will mess it up
2. names with equal length (or it doesn't line up) 

If you can suggest better names, I'm happy

Reply:
Those are good points, let's leave it as is then.